packages/features/audit-logs/lib/trpc-plugin.ts

```diff
@@ -0,0 +1,25 @@
+import type { UserFromSession } from "@calcom/trpc/server/middlewares/sessionMiddleware";
```

Comment:
Will be used in all trpc endpoints that perform an action that must be reported. Design is not finalized as edge cases arise it will be modified. Should be done in a couple of days